### PR TITLE
Mobile modifier

### PR DESCRIPTION
**Pull request by saschaben**
refines the layout, spacing, and consistency of the nav menu bar in large and small formats.

### CODE REVIEW

File: _sass/_layout.scss [hunk 1]
```diff
@@ -57,12 +57,19 @@
 
         // Gaps between nav items, but not on the first one
         &:not(:first-child) {
-            margin-left: 20px;
+            margin-left: 12px;
         }
 
         &.home-link {
```
Comment (jrjohnson) [inline _sass/_layout.scss]: This should get removed. We're not using it anymore.

**Comment by jrjohnson** [comment]
Agreed. It works, but not well. Can be substantially simplified.

File: _sass/_layout.scss (hunk 1 above)
Comment (saschaben) [reply _sass/_layout.scss]: done.

Comment (saschaben) [reply _sass/_layout.scss]: actually, it turns out we are. I've reinstated the style. we should discuss how to best approach cleaning up CSS / SCSS here.

Comment (jrjohnson) [reply _sass/_layout.scss]: I just added `&.home-link` yesterday and it is only used in one place and was replaced with `space-link` in this change.  So it shouldn't be used anywhere else.

Comment (saschaben) [reply _sass/_layout.scss]: space-link does not replace home-link; it provides a different case state for use with page-link. they are both still in use separately in the nav.

**Pull request merged**